gnome/gingerblue/Portfile

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gingerblue
+version             2.0.1
+revision            0
+set branch          [join [lrange [split $version .] 0 1] .]
+
+categories          gnome
+platforms           darwin
+license             GPL-3+
+maintainers         {gnome.org:ole @oleaamot} \
+                    openmaintainer
+description         GNOME Gingerblue
+long_description    Locate Internet Radio Stations
+homepage            https://www.gingerblue.org/
+master_sites        gnome:sources/${name}/${branch}/
+
+use_xz              yes
+
+checksums           rmd160  d1482a8ac4b1fd863f1f4edc267cc7e87e00a49f \
+                    sha256  118dc659d229e2fa7e6c03c8b7aef738f0e9e7e773cc941329b5afbeb82e88a9 \
+                    size    320484
+
+depends_build       port:autoconf \
+                    port:automake \
+                    port:geocode-glib \
+                    port:geoclue2 \
+                    port:gnome-common \
+                    port:gstreamer1-gst-plugins-bad \
+                    port:gstreamer1-gst-plugins-good \
+                    port:gstreamer1-gst-plugins-ugly \
+                    port:gtk-doc \
+                    port:intltool \
+                    port:itstool \
+                    port:pkgconfig \
+                    port:yelp-tools
+
+depends_lib         path:lib/pkgconfig/glib-2.0.pc:glib2 \
+                    path:lib/pkgconfig/pango.pc:pango \
+                    port:desktop-file-utils \
+                    port:geocode-glib \
+                    port:geoclue2 \
+                    port:gstreamer1 \
+                    port:gstreamer1-gst-plugins-base \
+                    port:gtk3 \
+                    port:libchamplain \
+                    port:libxml2 \
+                    port:zlib
+
+depends_run         port:adwaita-icon-theme \
+                    port:gstreamer1-gst-plugins-bad \
+                    port:gstreamer1-gst-plugins-good \
+                    port:gstreamer1-gst-plugins-ugly
+
+# reconfigure using autogen.sh from upstream git for intltool 0.51 compatibility
+
+configure.cmd       ./configure
```

Comment:
the comment and and the `configure.cmd` seem redundant, isn't `configure` the default command to do in the configure phase?

Reply:
I have removed the comment and the configure.cmd lines from the latest fork repository.

Reply:
thanks, can you please squash your commits?

Reply:
@oleaamot Unresolving this - can you please squash the 2 commits into a single one with the comment `gingerblue: new port`, as per @reneeotten's request?

Other than that I think this is good to go.